src/olympia/files/tests/test_models.py

```diff
@@ -257,7 +257,7 @@ def test_generate_filename_platform_specific(self):
 
     def test_generate_filename_many_apps(self):
         f = File.objects.get(id=67442)
-        f.version.compatible_apps = (amo.FIREFOX, amo.THUNDERBIRD)
+        f.version.compatible_apps = (amo.THUNDERBIRD, amo.FIREFOX)
```

Comment:
Yes. If the update test is executed without sorting code, it will append `tb+fx` to the file name. Now because of sorting it will append `fx+tb` to the code.
I just changed the order of compatible_apps in the test case to check if sorting is working fine.